Source/Core/VideoBackends/Software/EfbInterface.cpp

```diff
@@ -562,26 +557,7 @@ void CopyToXFB(yuv422_packed* xfb_in_ram, u32 fbWidth, u32 fbHeight, const EFBRe
 // main memory or doing a yuyv conversion
 void BypassXFB(u8* texture, u32 fbWidth, u32 fbHeight, const EFBRectangle& sourceRc, float Gamma)
 {
-  if (fbWidth * fbHeight > MAX_XFB_WIDTH * MAX_XFB_HEIGHT)
-  {
-    ERROR_LOG(VIDEO, "Framebuffer is too large: %ix%i", fbWidth, fbHeight);
-    return;
-  }
-
-  size_t textureAddress = 0;
-  const int left = sourceRc.left;
-  const int right = sourceRc.right;
-
-  for (u16 y = sourceRc.top; y < sourceRc.bottom; y++)
-  {
-    for (u16 x = left; x < right; x++)
-    {
-      const u32 color = Common::swap32(GetColor(x, y) | 0xFF);
-
-      std::memcpy(&texture[textureAddress], &color, sizeof(u32));
-      textureAddress += sizeof(u32);
-    }
-  }
+  // TODO: Upload directly to texture cache.
```

Comment:
Ah ok.  Come to think of it, I might have deleted the UI option as well.  I'll remove this if you think it isn't needed.